modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/clientConfig/CXFDefaultClientConfigurationTestCase.java

```diff
@@ -119,27 +115,31 @@ public void testDefaultClientConfigurationOnDispatchInContainer() throws Excepti
    @RunAsClient
    @OperateOnDeployment(CLIENT_DEP)
    public void testCustomClientConfigurationInContainer() throws Exception {
+      startContainerAndDeploy();
       assertEquals("1", runTestInContainer("testCustomClientConfiguration"));
    }
    
    @Test
    @RunAsClient
    @OperateOnDeployment(CLIENT_DEP)
    public void testCustomClientConfigurationOnDispatchInContainer() throws Exception {
+      startContainerAndDeploy();
       assertEquals("1", runTestInContainer("testCustomClientConfigurationOnDispatch"));
    }
    
    @Test
    @RunAsClient
    @OperateOnDeployment(CLIENT_DEP)
    public void testCustomClientConfigurationUsingFeatureInContainer() throws Exception {
+      startContainerAndDeploy();
       assertEquals("1", runTestInContainer("testCustomClientConfigurationUsingFeature"));
    }
    
    @Test
    @RunAsClient
    @OperateOnDeployment(CLIENT_DEP)
    public void testCustomClientConfigurationOnDispatchUsingFeatureInContainer() throws Exception {
+      startContainerAndDeploy();
```

Comment:
@gaol  Just some minor things:  Could you please add some TODO comment to reminds us to restore this @Before method after this issue is fixed in ARQ ?